lib/godmin/helpers/forms.rb

```diff
@@ -4,6 +4,11 @@ module Forms
       def form_for(record, options = {}, &block)
         super(record, { builder: FormBuilders::FormBuilder, inline_errors: false }.merge(options), &block)
       end
+
+      def select_tag(name, option_tags = nil, options = {})
```

Comment:
Got you, wasn't aware of this helper. It seems to piggy back on a defined form builder, but passing it a (fake) string object.

Would you find this sort of solution acceptable?

```ruby
# [2] pry(#<#<Class:0x00561108c1eb88>>)> $ bootstrap_form_tag

# From: /cache/gems/2.2.3/gems/bootstrap_form-2.2.0/lib/bootstrap_form/helper.rb  @ line 29:
# Owner: BootstrapForm::Helper
# Visibility: public
# Number of lines: 5

def bootstrap_form_tag(options = {}, &block)
  options[:acts_like_form_tag] = true

  bootstrap_form_for("", options, &block)
end
# [3] pry(#<#<Class:0x00561108c1eb88>>)> $ bootstrap_form_for

# From: /cache/gems/2.2.3/gems/bootstrap_form-2.2.0/lib/bootstrap_form/helper.rb  @ line 7:
# Owner: BootstrapForm::Helper
# Visibility: public
# Number of lines: 21

def bootstrap_form_for(object, options = {}, &block)
  options.reverse_merge!({builder: BootstrapForm::FormBuilder})

  options[:html] ||= {}
  options[:html][:role] ||= 'form'

  layout = case options[:layout]
    when :inline
      "form-inline"
    when :horizontal
      "form-horizontal"
  end

  if layout
    options[:html][:class] = [options[:html][:class], layout].compact.join(" ")
  end

  temporarily_disable_field_error_proc do
    form_for(object, options, &block)
  end
end

```